content/docs/hosting/BYOC.md

```diff
@@ -98,169 +99,178 @@ With Appbase.io Dashboard you can easily share cluster with your team members. E
 
 ---
 
-### Using AMI
-
-Arc is also available via an Amazon Machine Image (AMI) on the AWS Marketplace. Arc works running a `systemd` service. With the AMI, you can seamlessly deploy Arc on an AWS EC2 instance.
+### How to create Arc instance?
```

Comment:
The section title isn't right. This section just describes the process to get the ARC_ID. It's unrelated to creation of Arc instance.

Reply:
@siddharthlatest I used title based on what we have on Arc Dashboard, can you suggest better title?